sdk/keyvault/azure-keyvault-certificates/dev_requirements.txt

```diff
@@ -3,5 +3,6 @@
 -e ../../../tools/azure-sdk-tools
 -e ../../identity/azure-identity
 -e ../azure-mgmt-keyvault
+../azure-keyvault-nspkg
```

Comment:
missing a -e ?

Reply:
It turns out an editable install isn't required to pass CI, so I removed the -e.